### PR TITLE
Update Vectorscan with Snort 3 Learning Path

### PR DESCRIPTION
**Pull request by annietllnd**
- Vectorscan doesn't need to be modified anymore for cross-compilation
- Version and link updates

Before submitting a pull request for a new Learning Path, please review [Create a Learning Path](https://learn.arm.com//learning-paths/cross-platform/_example-learning-path/)
- [x] I have reviewed Create a Learning Path

Please do not include any confidential information in your contribution. This includes confidential microarchitecture details and unannounced product information. No AI tool can be used to generate either content or code when creating a learning path or install guide.

- [x] I have checked my contribution for confidential information

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the Creative Commons Attribution 4.0 International License.

### CODE REVIEW

File: content/learning-paths/servers-and-cloud-computing/vectorscan/install.md
Comment (pareenaverma) [inline content/learning-paths/servers-and-cloud-computing/vectorscan/install.md:1]: @annietllnd we should retain the part about vectorscan on hardware with/without SVE. To build with use use BUILD_SVE with cmake

Comment (annietllnd) [reply content/learning-paths/servers-and-cloud-computing/vectorscan/install.md:1]: Great catch, done

File: content/learning-paths/servers-and-cloud-computing/vectorscan/snort.md [hunk 1]
```diff
@@ -135,9 +135,9 @@ You can now download, compile and build Snort 3:
 
 ```bash
 cd ~/snort_src
-wget https://github.com/snort3/snort3/archive/refs/tags/3.1.18.0.tar.gz -O snort3-3.1.18.0.tar.gz
-tar -xzvf snort3-3.1.18.0.tar.gz
-cd snort3-3.1.18.0
+wget https://github.com/snort3/snort3/archive/refs/tags/3.2.2.0.tar.gz -O snort3-3.2.2.0.tar.gz
+tar -xzvf snort3-3.2.2.0.tar.gz
+cd snort3-3.2.2.0
 ./configure_cmake.sh --prefix=/usr/local --enable-tcmalloc --enable-jemalloc
```
Comment (pareenaverma) [inline content/learning-paths/servers-and-cloud-computing/vectorscan/snort.md:141]: --enable-tcmalloc --enable-jemalloc are mutually exclusive. I'm removing --enable-jemalloc

File: content/learning-paths/servers-and-cloud-computing/vectorscan/snort.md [hunk 2]
```diff
@@ -135,9 +135,9 @@ You can now download, compile and build Snort 3:
 
 ```bash
 cd ~/snort_src
-wget https://github.com/snort3/snort3/archive/refs/tags/3.1.18.0.tar.gz -O snort3-3.1.18.0.tar.gz
-tar -xzvf snort3-3.1.18.0.tar.gz
-cd snort3-3.1.18.0
+wget https://github.com/snort3/snort3/archive/refs/tags/3.2.2.0.tar.gz -O snort3-3.2.2.0.tar.gz
```
Comment (pareenaverma) [inline content/learning-paths/servers-and-cloud-computing/vectorscan/snort.md:138]: I'll update the output from snort to reflect the newer version

**Pull request merged**